test/internal/preferences/preferences.test.ts

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I would recommend _not_ ending this file name with `.test.ts`, since it has no tests and you do not want it getting loaded by mocha.

Reply:
good point